Source/WebCore/platform/graphics/cg/ColorSpaceCG.cpp

```diff
@@ -40,7 +40,7 @@ template<const CFStringRef& colorSpaceNameGlobalConstant> static CGColorSpaceRef
     static NeverDestroyed<RetainPtr<CGColorSpaceRef>> colorSpace;
     static std::once_flag onceFlag;
     std::call_once(onceFlag, [] {
-        colorSpace.get() = adoptCF(CGColorSpaceCreateWithName(colorSpaceNameGlobalConstant));
+        colorSpace.get() = adoptCF(CGColorSpaceCreateWithName(RetainPtr { colorSpaceNameGlobalConstant }.get()));
```

Comment:
`NeverDestroyed` is designed for immediate initialization. Initializing to null and then assigning later is bad code smell. To avoid the `.get() =` (which we maybe need to prohibit eventually), use `LazyNeverDestroyed` and call `construct`.